src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationWriter.cs

```diff
@@ -477,21 +477,17 @@ protected void WriteStartElement(string name, string? ns, object? o, bool writeP
             bool needEmptyDefaultNamespace = false;
             if (_namespaces != null)
             {
-                foreach (string alias in _namespaces.Namespaces.Keys)
-                {
-                    string? aliasNs = (string?)_namespaces.Namespaces[alias];
+                _namespaces.TryLookupPrefix(ns, out prefix);
 
-                    if (alias.Length > 0 && aliasNs == ns)
-                        prefix = alias;
-                    if (alias.Length == 0)
-                    {
-                        if (aliasNs == null || aliasNs.Length == 0)
-                            needEmptyDefaultNamespace = true;
-                        if (ns != aliasNs)
-                            writePrefixed = true;
-                    }
+                if (_namespaces.TryLookupNamespace("", out string? defaultNS))
```

Comment:
Use **string.Empty**?

Reply:
We tend to use "" throughout serializer code when referencing the default namespace. I believe the compiler reconciles them both to the same thing anyway. But for consistency, I'm sticking with "".